### ISSUE TITLE
Timer issues in incognito mode

### ISSUE THREAD

**Issue by randomnameletsgo** [proginosko/LeechBlockNG-chrome#78]
Not sure if this counts as a feature request or not, or if it even is an intended feature, but here does
I often watch youtube in incognito mode, as I dont want to contaminate my reccomended with say, music videos
Ive set a limit on youtube of 20 minutes in every 2 hours
Anyways, I noticed that I had about 6 minutes left in the incognito window when I closed it. And when I opened yt on my main, it was back to 18 minutes, and when I opened incognito again, it also had 18 minutes, thereby making an infinite loop where I never run out of time if I just keep switching the windows
Its also somewhat random, as sometimes I close and open incognito and its still at the 7 minutes where I left it before, and sometimes its back to the 20 minutes Ive set it to. Today, my window was blocked due to hitting the limit, but after reopening incognito it was unlocked one time, but still blocked another time I tested it, and its just confusing
Sorry for my bad english, please fix

**Comment by proginosko** [comment]
Yes, this is a known issue due to the fact that the incognito window runs a separate instance of the extension (see [here](https://developer.chrome.com/docs/extensions/reference/manifest/incognito) under "Split mode") which means that the time data can get out of sync between the regular window and the incognito window. For now, the workaround would be to close all browser windows before switching between modes.

Another workaround would be to use two block sets: one for non-private tabs and the other for private (incognito) tabs.

**Comment by randomnameletsgo** [comment]
Ahh, thanks for the clarification
